lib/sentry/config.ex

```diff
@@ -147,6 +147,14 @@ defmodule Sentry.Config do
       [`:jason`](https://hex.pm/packages/jason) as a dependency of your application.
       """
     ],
+    send_client_reports: [
+      type: :boolean,
+      default: true,
+      doc: """
+      Send SDK Client Reports, this option defaults to `true`
+      *Available since v10.0.0*.
```

Comment:
1. nimble_options already mentions the default value in the generated doc.
2. This is not available since 10.0.0, but rather will be available from 10.8.0.
3. Let's link to Client Reports documentation.